src/axom/sidre/spio/interface/spio_shroud.yaml

```diff
@@ -27,25 +27,21 @@ typemap:
 - type: axom::sidre::DataStore
   fields:
     base: shadow
-    cxx_header: axom/sidre/core/DataStore.hpp
-#    cxx_type: DataStore
+    wrap_header: axom/sidre/interface/c_fortran/wrapDataStore.h
```

Comment:
Is this telling it what file to put the wrap header into (i.e. overriding the default). ?

Reply:
No, this is telling it where the generated wrapped headers are located and used with wrapped classes since they generate a struct for each class (the shadow type).  The intent of this file is to allow one wrapped library to be used by another wrapped library.  It's not being used yet but I think there is a use case.  Currently spio_shroud.yaml duplicates some of the information in the file since it uses sidre wrapped types.  But eventually there should be some sort of include facility in Shroud.